documentation/src/docs/getting-started.md

```diff
@@ -105,5 +105,5 @@ No more events from examples/helloStream
 ...
 ```
 
-See the readme.md file in the pravega-client-examples for more details on running the
+See the [readme.md](https://github.com/pravega/pravega-samples/blob/v0.3.0/pravega-client-examples/README.md) file in the pravega-client-examples for more details on running the
```

Comment:
`README.md` capital to be consistent with the previous ones.

Reply:
done.